spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe "Ingredients Index", type: :feature do
+  describe "When I visit /ingredients" do
+    it "I see a list of all ingredients including their name and cost" do
+      ingredient_1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+      ingredient_2 = Ingredient.create!(name: "Salt", cost: 4)
+
+      visit "/ingredients"
+
+      expect(page).to have_content("#{ingredient_1.name}: #{ingredient_1.cost}")
```

Comment:
I really like the way you interpolated the attributes into that long string to make sure the cost is associated with the right ingredient! Makes this test more robust 👍